recipe-appTests/listRecipesTest.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  listRestaurantsTest.swift
+//  recipe-appTests
+//
+//  Created by Arthur Hermann on 27/11/2024.
+//
+
+import XCTest
+import Testing
+@testable import recipe_app
+
+final class listRestaurantsTest: XCTestCase {
+    
+    func testRecipeListNotEmpty() {
+        DispatchQueue.global(qos: .background).async {
```

Comment:
You're using `DispatchQueue.global(qos: .background).async`, but it doesn’t wait for the operation to finish before the test completes. This could cause the tests to give false results if the network call hasn’t finished yet.

`Suggestion:` Use `XCTestExpectation` instead. It’s a better way to handle async operations in tests because it ensures the test waits for everything to finish properly.